modules/slugs/admin/slugs-admin.php

```diff
@@ -112,19 +112,13 @@ function qtranxf_slugs_deactivate() {
  * Creates a metabox for every post type available.
  */
 function qtranxf_slugs_add_slug_meta_box() {
-    global $post_type;
-
-    // Exclude some WooCommerce shop pages
-    $not_applicable_types = [
-        'shop_coupon',
-        'shop_order',
-    ];
-    if ( isset( $post_type ) && in_array( $post_type, $not_applicable_types ) ) {
-        return;
-    }
+    global $wp_meta_boxes;
 
-    remove_meta_box( 'slugdiv', null, 'normal' );
-    add_meta_box( 'qts_sectionid', __( 'Slugs per language', 'qtranslate' ), 'qtranxf_slugs_draw_meta_box', null, 'side', 'high' );
+    //Replace slugs metabox only if existing and not already removed
+    if ( ! empty($wp_meta_boxes[ get_current_screen()->id ][ 'normal' ][ 'core' ][ 'slugdiv' ] ) ) {
```

Comment:
I can't test at this moment. Is this robust enough if one key is missing? I'm afraid `empty` will raise an exception if one key is missing - unlike `unset`.

Reply:
Based on php.net,

> empty() is essentially the concise equivalent to !isset($var) || $var == false

which is what we need here, because that key may not be existing or may be `false` if already removed, so we should be covered.